### PR TITLE
[ADD] middleware for pyramid framework

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

File: src/pybrake/middleware/pyramid.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+        headers=dict(request.headers),
+        environ=request.environ,
+        view_args=request.urlargs,
+    )
+
+    return notice
+
+
+def handle_exception(notifier, exception):
+    notice = notifier.build_notice(exception)
```
Comment (kyrylo) [inline src/pybrake/middleware/pyramid.py:80]: Why not just `notifier.send_notice(exception)`?

Comment (kyrylo) [inline src/pybrake/middleware/pyramid.py:80]: Why not just `notifier.send_notice(exception)`?

**Review by kyrylo — APPROVED** [review]

File: src/pybrake/middleware/pyramid.py (hunk 1 above)
Comment (smurf-U) [reply src/pybrake/middleware/pyramid.py:80]: `send_notice` in Pybrake accepts `dict` objects made by `build_notice` until now. We can supply an instance of `exception` directly to `send_notice`, but we must first update the `Notifier` class.

We should, in my opinion, initiate a new PR for that.

Comment (kyrylo) [reply src/pybrake/middleware/pyramid.py:80]: Our README states we can pass the exception directly: https://github.com/airbrake/pybrake#sending-errors-to-airbrake 

Should that be updated?

Comment (smurf-U) [reply src/pybrake/middleware/pyramid.py:80]: PTAL

Comment (smurf-U) [reply src/pybrake/middleware/pyramid.py:80]: PTAL

**Review by kyrylo — APPROVED** [review]

**Pull request merged**